pywr/nodes.py

```diff
@@ -495,6 +495,9 @@ class RollingVirtualStorage(
 ):
     """A rolling virtual storage node useful for implementing rolling licences.
 
+    If the initial volume of the storage is less than the maximum volume then the parameter will calculate
+    an initial utilisation value. This is set equal to: (max volume - initial volume) / timesteps - 1.
```

Comment:
I think this needs to explain a bit more clearly that the initial utilisation is assumed to have happened equally over a number of periods prior to the model start. I.e. the licence will replenish at this rate until a full period is complete.

Reply:
Should we also note the change in behaviour?

Reply:
I've expanded the docstring a bit